Packages/MIES/MIES_Async.ipf

```diff
@@ -1031,3 +1036,24 @@ static Function/WAVE GetWorkloadOrder(dfr)
 	SetDimLabel 1, 1, $"orderGlobal", wv
 	return wv
 End
+
+/// @brief returns 1 if ASYNC framework is running, 0 otherwise
+static Function IsASYNCRunning()
+
+	variable waitResult, err, doe
+
+	NVAR tgID = $GetThreadGroupID()
+	try
+		DebuggerOptions
```

Comment:
Please use the dedicated helpers: `ResetDebugOnError` and `DisableDebugOnError`.

Reply:
done